src/projects/diff/hunk-sorter.html

```diff
@@ -159,6 +170,11 @@
         if (unsortedGroup.diff.length > 0) {
           allGroups.push(unsortedGroup);
         }
+        if (fbOrdering.groups === undefined && this.backendPath && defaultGroup.diff.length > 0) {
+          this.$.checkIfPathIsThere.body = defaultGroup;
```

Comment:
Unsorted changes always come from GitHub so I think we should just change the `Unsorted changes` string